tests/page/page-route.spec.ts

```diff
@@ -837,3 +837,76 @@ for (const method of ['fulfill', 'continue', 'abort'] as const) {
     expect(e.message).toContain('Route is already handled!');
   });
 }
+
+it('should chain continue', async ({ page, server }) => {
+  const intercepted = [];
+  await page.route('**/empty.html', route => {
+    intercepted.push(1);
+    route.continue();
+  });
+  await page.route('**/empty.html', route => {
+    intercepted.push(2);
+    route.continue();
+  });
+  await page.route('**/empty.html', route => {
+    intercepted.push(3);
+    route.continue();
+  });
+  await page.goto(server.EMPTY_PAGE);
+  expect(intercepted).toEqual([3, 2, 1]);
+});
+
+it('should not chain fulfill', async ({ page, server }) => {
+  await page.route('**/empty.html', route => {
+    route.continue();
```

Comment:
throw here or asserting something?

Reply:
Sure. I'm doing that in a browser context code, the same code serves it.